skpy/event.py

```diff
@@ -114,7 +118,11 @@ def rawToFields(cls, raw={}):
         fields.update({"userId": SkypeUtils.userToId(res.get("selfLink")),
                        "name": res.get("privateInfo", {}).get("epname"),
                        "capabilities": list(filter(None, res.get("publicInfo", {})
-                                                            .get("capabilities", "").split(" | ")))})
+                                                            .get("capabilities", "").split(" | "))),
+                       "version": res.get("publicInfo", {})
+                                            .get("skypeNameVersion",{}),
+                       "typ": res.get("publicInfo", {})
+                                 .get("typ",{})})
```

Comment:
You're defaulting both `version` and `typ` to an empty dict when not present -- these should be `None` (by omitting the default argument).